config/ompi_setup_prrte.m4

```diff
@@ -150,8 +150,8 @@ AC_DEFUN([_OMPI_SETUP_PRRTE_INTERNAL], [
     AS_IF([test -n "$prte_prefix_by_default" -a -n "$deprecated_prefix_by_default"],
           [AC_MSG_ERROR([--enable-prte-prefix-by-default cannot be used with --enable-mpirun-prefix-by-default or --enable-orterun-prefix-by-default.  Please only specify --enable-prte-prefix-by-default.])])
 
-    internal_prrte_CPPFLAGS=
-    internal_prrte_args="--with-proxy-version-string=$OPAL_VERSION --with-proxy-package-name=\"Open MPI\" --with-proxy-bugreport=\"https://www.open-mpi.org/community/help/\""
+    internal_prrte_CPPFLAGS="-I$OMPI_TOP_BUILDDIR/3rd-party/openpmix"
+    internal_prrte_args="--with-pmix-incdir=$OMPI_TOP_BUILDDIR/3rd-party/openpmix/src/include --with-proxy-version-string=$OPAL_VERSION --with-proxy-package-name=\"Open MPI\" --with-proxy-bugreport=\"https://www.open-mpi.org/community/help/\""
```

Comment:
You should just add this -I in the CPPFLAGS on the line above.  That might break things right now, so we may not be able to bump the submodule pointers right now, but we're working on that as the right fix.

`--with-pmix-incdir` will be removed shortly from PRRTE master, so don't use it.

Reply:
This should be all you need to use PMIx master as of 0372d50 and PRRTE once https://github.com/openpmix/prrte/pull/1239 is committed so that the FT bits compile again.

```
--- a/config/opal_config_pmix.m4
+++ b/config/opal_config_pmix.m4
@@ -251,7 +251,7 @@ dnl Expectation is that this is called only if external fails, the
 dnl caller configured libpmix configure, and the configure script
 dnl succeeded.
 AC_DEFUN([_OPAL_CONFIG_PMIX_INTERNAL_POST], [
-    opal_pmix_CPPFLAGS="-I$OMPI_TOP_BUILDDIR/3rd-party/openpmix/include -I$OMPI_TOP_SRCDIR/3rd-party/openpmix/include"
+    opal_pmix_CPPFLAGS="-I$OMPI_TOP_BUILDDIR/3rd-party/openpmix/include -I$OMPI_TOP_SRCDIR/3rd-party/openpmix/include -I$OMPI_TOP_BUILDDIR/3rd-party/openpmix/ -I$OMPI_TOP_SRCDIR/3rd-party/openpmix/"
     opal_pmix_LDFLAGS=""
     opal_pmix_LIBS="$OMPI_TOP_BUILDDIR/3rd-party/openpmix/src/libpmix.la"
     opal_pmix_WRAPPER_LIBS="-lpmix $opal_hwloc_WRAPPER_LIBS $opal_libevent_WRAPPER_LIBS"
```

Reply:
Thanks - updated the commit to reflect this.